kickstarter.Rmd

```diff
@@ -126,11 +126,64 @@ catOverTimePlot <- ggplot(
 
 print(catOverTimePlot)
 
-# Density plot of projects
+# Unfortunately this isn't super readable?
```

Comment:
This was a note to myself regarding the line plot, which was my first attempt to create a visualization to answer the question of which project categories were most popular over time.